mir_connector/inorbit_mir_connector/src/connector.py

```diff
@@ -271,3 +290,122 @@ def _execution_loop(self):
             self.mission_tracking.report_mission(self.status, self.metrics)
         except Exception:
             self._logger.exception("Error reporting mission")
+
+    def send_waypoint_over_missions(self, pose):
+        """Use the connector's mission group to create a move mission to a designated pose."""
+        mission_id = str(uuid.uuid4())
+        connector_type = self.config.connector_type
+        firmware_version = self.config.connector_config.mir_firmware_version
+
+        self.mir_api.create_mission(
+            group_id=self.tmp_missions_group_id,
+            name="Move to waypoint",
+            guid=mission_id,
+            description="Mission created by InOrbit",
+        )
+        param_values = {
+            "x": float(pose["x"]),
+            "y": float(pose["y"]),
+            "orientation": math.degrees(float(pose["theta"])),
```

Comment:
should you convert theta from radians here?

Reply:
Yes. Theta is received in radians from inorbit and must be converted to degrees. You can see the same operation being done in the previous implementation by the API class [here](https://github.com/inorbit-ai/inorbit-robot-connectors/blob/226bbdd627c20745827c4f3611adb9b1e163ba9e/mir_connector/inorbit_mir_connector/src/mir_api/mir_api_v2.py#L183).